### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by wibily**
### What this PR does

Adds language translation toggle to enable switching between languages

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [ x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/ReactViews/Mobile/MobileMenuItem.jsx [hunk 1]
```diff
@@ -34,7 +37,9 @@ export default function MobileMenuItem(props) {
 MobileMenuItem.propTypes = {
   href: PropTypes.string,
   onClick: PropTypes.func,
-  caption: PropTypes.string
+  caption: PropTypes.string,
+  showMobileIcon: PropTypes.bool,
```
Comment (nf-s) [inline lib/ReactViews/Mobile/MobileMenuItem.jsx]: Is `showMobileIcon` used anywhere?

**Review by nf-s — CHANGES_REQUESTED** [review]
This isn't super high priority - but if it isn't too much work - it would be nice if the wrapping was a bit better on small screens (before transition to mobile screen):

## With language switcher 

image

## Without language switcher

image

File: lib/ReactViews/Mobile/MobileMenuItem.jsx [hunk 2]
```diff
@@ -34,7 +37,9 @@ export default function MobileMenuItem(props) {
...
+  icon: PropTypes.string
```
Comment (nf-s) [inline lib/ReactViews/Mobile/MobileMenuItem.jsx]: I think this should be `PropTypes.object` as you are passing in an object here

https://github.com/TerriaJS/terriajs/blob/d1515b43b0c24350717699c0ac056a3d0d6f9484/lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx#L31-L43

Comment (nf-s) [reply lib/ReactViews/Mobile/MobileMenuItem.jsx]: Which gets passed in here
https://github.com/TerriaJS/terriajs/blob/d1515b43b0c24350717699c0ac056a3d0d6f9484/lib/ReactViews/Map/Panels/MobilePanel.jsx#L27-L31

File: lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+              <RawButton onClick={() => i18n.changeLanguage(key)}>
+                {value}
+              </RawButton>
+            </Li>
+          ))}
+        </Ul>
+      </Box>
+    </MenuPanel>
+  );
+};
```
Comment (nf-s) [inline lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx:59]: I'm getting this error, but not sure what it's about 
image

File: lib/ReactViews/Mobile/MobileMenuItem.jsx (hunk 1 above)
Comment (wibily) [reply lib/ReactViews/Mobile/MobileMenuItem.jsx]: ah, good catch

File: lib/ReactViews/Mobile/MobileMenuItem.jsx (hunk 2 above)
Comment (wibily) [reply lib/ReactViews/Mobile/MobileMenuItem.jsx]: sorry i don't understand, are you suggesting that MobileMenuItem should have a single prop with the caption and the icon as keys?

File: lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx (hunk 3 above)
Comment (wibily) [reply lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx:59]: will investigate

File: lib/ReactViews/Mobile/MobileMenuItem.jsx (hunk 2 above)
Comment (nf-s) [reply lib/ReactViews/Mobile/MobileMenuItem.jsx]: Sorry I meant `icon` should be `PropTypes.object` - as you are passing in an `object` through:
- `MenuPanel` `mobileIcon` prop -> `MobileMenuItem` `icon` prop

I was getting an error in console about the prop being an `object`, even though it's prop type is `string`

File: lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx (hunk 3 above)
Comment (wibily) [reply lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx:59]: hmm, i couldn't reproduce this one

Comment (nf-s) [reply lib/ReactViews/Map/Panels/LangPanel/LangPanel.tsx:59]: i can't either!
it has disappeared

**Review by nf-s — APPROVED** [review]

**Pull request merged**